src/managers/UserManager.js

```diff
@@ -37,6 +39,10 @@ class UserManager extends BaseManager {
   resolve(user) {
     if (user instanceof GuildMember) return user.user;
     if (user instanceof Message) return user.author;
+    if (typeof user === 'string') {
+      const match = user.match(USERS_PATTERN)
+      if (match !== null) user = userId;
```

Comment:
```suggestion
      const match = user.match(USERS_PATTERN);
      if (match !== null) user = match[0];
```
Seems like you messed these up, gotta also make ESLint happy.

Reply:
Fixed in latest commit, thanks :)